packages/create-block-quick-start-template/block-templates/save.js.mustache

```diff
@@ -0,0 +1,36 @@
+/**
+ * React hook that is used to mark the block wrapper element.
+ * It provides all the necessary props like the class name.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/#useblockprops
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * The save function defines the way in which the different attributes should
+ * be combined into the final markup, which is then serialized by the block
+ * editor into `post_content`.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-edit-save/#save
+ *
+ * @param {Object} props            Properties passed to the function.
+ * @param {Object} props.attributes Available block attributes.
+ * 
+ * @return {Element} Element to render.
+ */
+export default function Save( { attributes } ) {
+	const { showStartingYear, startingYear } = attributes;
+
+	// Get the current year.
+	const currentYear = new Date().getFullYear();
```

Comment:
This is not going to work as expected in the long run. The result of calling the `save` function needs always to return the same result. In this case, the year changes every year. In effect, if someone saves the content in 2023, if they open the same block in 2024 in the block editor, the `save` will produce a different HTML, causing the editor to show a validation error.